v0/destinations/google_cloud_function/util.js

```diff
@@ -0,0 +1,63 @@
+const {
+    defaultBatchRequestConfig
+} = require("../../util");
+
+const {
+    TRIGGERTYPE
+} = require("./config");
+
+function generateBatchedPayloadForArray(events) {
+    const batchResponseList = [];
+    const metadata = [];
+
+    // extracting destination
+    // from the first event in a batch
+    const {
+        destination
+    } = events[0];
+    const Config = destination.Config;
+    const {
+        googleCloudFunctionUrl,
+        triggerType,
+        apiKeyId,
+        gcloudAuthorization
+    } = Config;
+    let batchEventResponse = defaultBatchRequestConfig();
+
+    // Batch event into dest batch structure
+    events.forEach(ev => {
+        batchResponseList.push(ev.message.body.JSON);
+        metadata.push(ev.metadata);
+    });
+
+    batchEventResponse.batchedRequest.body.JSON_ARRAY = {
```

Comment:
@utsabc how we are planning to send the batched payloads? in aws lambda we have sent is JSON block within the `payload` array. Should we do that here also?